templates/web/fixmystreet.com/about/council.html

```diff
@@ -4,66 +4,102 @@
         <div class="councils-features__feature">
-          <div class="councils-features__thumb councils-features__thumb--customise"></div>
-          <h3>Stay flexible</h3>
-          <p>FixMyStreet is open standards compliant. That gives you complete customer flexibility to work with additional contractors at any time. Integrate other services as and when you need them, and provide additional functionalities to your residents.</p>
+          <div class="councils-features__thumb councils-features__thumb--brand"></div>
+          <h3>Our experience, your brand</h3>
+          <p>White-labelled product, branded to complement your existing services.</p>
+        </div>
+        <div class="councils-features__feature">
+          <div class="councils-features__thumb councils-features__thumb--end-to-end"></div>
+          <h3>End-to-end case management</h3>
+          <p>A complete case management solution for customer services, inspection and maintenance teams.</p>
+        </div>
+        <div class="councils-features__feature">
+          <div class="councils-features__thumb councils-features__thumb--intelligent-routing"></div>
+          <h3>Intelligent routing</h3>
+          <p>Send the problem to the correct individual,
+          team or another authority based upon location, category or a combination of both.</p>
+        </div>
+        <div class="councils-features__feature">
+          <div class="councils-features__thumb councils-features__thumb--automated-instruction"></div>
+          <h3>Automated instruction</h3>
+          <p>Route problems to contracted providers based on category, location or meta data.</p>
         </div>
         <div class="councils-features__feature">
-          <div class="councils-features__thumb councils-features__thumb--bringyourown"></div>
-          <h3>Bring your own</h3>
-          <p>You’ve already invested in mapping, asset, customer and case management services. FixMyStreet slots right in with your existing portfolio, so you can continue to use the systems your team knows and trusts.</p>
+          <div class="councils-features__thumb councils-features__thumb--authority"></div>
+          <h3>Unitary and two-tier authorities</h3>
+          <p>Handle all reports or a subset of reports by category, location or a combination of both.</p>
         </div>
         <div class="councils-features__feature">
-          <div class="councils-features__thumb councils-features__thumb--process"></div>
-          <h3>Streamline your process </h3>
-          <p>FixMyStreet is the first case-management service designed with councils, for councils. It streamlines your processes right through from customer service, to inspection, instruction and resolution.</p>
+          <div class="councils-features__thumb councils-features__thumb--syncronised"></div>
+          <h3>Synchronised with FixMyStreet.com</h3>
+          <p>Handle all reports or a subset of reports by category, location or a combination of both.</p>
```

Comment:
The text here is the same as for the previous feature, which I'm guessing is not intentional?